specs/networking/libp2p-standardization.md

```diff
@@ -39,9 +47,10 @@ Current defaults are:
 - Digests: `SHA256` (also supports `SHA512`)
 
 *Note: Secio is being deprecated in favour of [TLS
-1.3](https://github.com/libp2p/specs/blob/master/tls/tls.md). It is our
-intention to transition to use TLS 1.3 for encryption between nodes, rather
-than Secio.*
+1.3](https://github.com/libp2p/specs/blob/master/tls/tls.md) or
```

Comment:
Yep agreed. Perhaps we raise this in the call.

Reply:
This didn't come up in the call. But indeed, TLS 1.3 language support is not universal for now. For example, nodejs lacks such support. Protocol Labs is funding @jasnell's work on adding QUIC support to nodejs (https://github.com/nodejs/quic) + TLS 1.3, because it's a prerequisite.

The libp2p team is very favourable to Noise. I believe rust-libp2p introduced early experimental support (but hasn't submitted a spec to libp2p/specs yet), and we've toyed with the idea of adding support for go-libp2p. Here's a spec: https://github.com/libp2p/go-libp2p/issues/631

I think we should prioritise these efforts. I want to support the IK, IX and XX handshakes.

Reply:
An issue with Noise is the compatibility with QUIC. There's [nQUIC](https://eprint.iacr.org/2019/028.pdf), and I've asked @marten-seemann (our QUIC expert) to look into it this quarter.

Reply:
Considering QUIC is also not yet widely supported, I'm okay with Noise without QUIC for now with the intention to move to TLS 1.3 + QUIC as support becomes available.

I'd say getting a Noise spec written in libp2p/specs is a top priority at this point.

Reply:
I agree with this and will update the wording to reflect this above, unless there are any objections?

Reply:
Sounds good. Note that clients supporting QUIC will advertise so in their multiaddrs, so other clients/languages supporting that combo should immediately benefit from that enhanced transport. Clients/languages not supporting QUIC will simply not dial those addresses, and will instead fallback to TCP automatically.